### PR TITLE
feat(time): support convert to time

### PR DESCRIPTION
**Pull request by whalecold**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by whalecold** [comment]
@iawia002

File: service/content.go [hunk 1]
```diff
@@ -370,38 +371,40 @@ func (p *ContextPrefab) Make(ctx context.Context) (interface{}, error) {
...
+	reflect.TypeOf(int64(0)):       ConvertToInt64,
+	reflect.TypeOf(uint(0)):        ConvertToUint,
+	reflect.TypeOf(uint8(0)):       ConvertToUint8,
+	reflect.TypeOf(uint16(0)):      ConvertToUint16,
+	reflect.TypeOf(uint32(0)):      ConvertToUint32,
+	reflect.TypeOf(uint64(0)):      ConvertToUint64,
+	reflect.TypeOf(float32(0)):     ConvertToFloat32,
+	reflect.TypeOf(float64(0)):     ConvertToFloat64,
+	reflect.TypeOf(time.Time{}):    ConvertToTime,
+	reflect.TypeOf(""):             ConvertToString,
```
Comment (iawia002) [inline service/content.go:388]: 顺序调一下吧，后面 pointer 的部分 time 是在 string 后面的

File: utils/generators/swagger/generator.go [hunk 2]
```diff
@@ -498,38 +498,40 @@ func (g *Generator) generateAutoParameter(typ api.TypeName) []spec.Parameter {
...
+	"int64":      service.ConvertToInt64,
+	"uint":       service.ConvertToUint,
+	"uint8":      service.ConvertToUint8,
+	"uint16":     service.ConvertToUint16,
+	"uint32":     service.ConvertToUint32,
+	"uint64":     service.ConvertToUint64,
+	"float32":    service.ConvertToFloat32,
+	"float64":    service.ConvertToFloat64,
+	"time.Time":  service.ConvertToTime,
+	"string":     service.ConvertToString,
```
Comment (iawia002) [inline utils/generators/swagger/generator.go:515]: 👆 同上

Comment (whalecold) [reply utils/generators/swagger/generator.go]: done

File: service/content.go (hunk 1 above)
Comment (whalecold) [reply service/content.go]: done

**Pull request merged**